### PR TITLE
Shimmy/register modal style

### PR DESCRIPTION
**Pull request by hidesminimally**
:warning: Not ready

### CODE REVIEW

**Comment by shannonchu** [comment]
@andylouisqin ready to merge.

Some things that aren't working: 
- Text color and input field styling (borders and background) are being overridden by foundation; 
- When you click create account and the modal shows up, if you scroll down it shows the rest of the page when it's supposed to just stay fixed.  But when "position: fixed;" is added to the full-modal styling, when if you are scrolled down on the landing page and then click create account, the modal only shows partially.

File: app/assets/stylesheets/static_pages.scss [hunk 1]
```diff
@@ -4,6 +4,62 @@
...
+    padding-top: 45px;
+  }
+
+  .cancel {
+    float: right;
+    margin-left: 8px;
+  }
+
+  .modal-header h3 {
+    color: ashby-gray;
```
Comment (shannonchu) [inline app/assets/stylesheets/static_pages.scss]: This is not working :'(

File: app/assets/stylesheets/static_pages.scss [hunk 2]
```diff
@@ -4,6 +4,62 @@
...
+  }
+
+  .instructions h6 {
+    color: ashby-gray;
```
Comment (shannonchu) [inline app/assets/stylesheets/static_pages.scss]: dis too

File: app/assets/stylesheets/static_pages.scss [hunk 3]
```diff
@@ -4,6 +4,62 @@
...
+
+  .input-form {
+    float: left;
+    width: 200px;
+    border-bottom: 3 px solid ashby-gray;
+    border-top: none;
+    border-left: none;
+    border-right: none;
+    background: transparent;
+  }
```
Comment (shannonchu) [inline app/assets/stylesheets/static_pages.scss:59]: borders & background not rendering

File: app/assets/stylesheets/foundation_and_overrides.scss [hunk 4]
```diff
@@ -96,7 +96,7 @@ $column-gutter: rem-calc(30);
 // $font-weight-normal: normal;
 // $font-weight-bold: bold;
 
-// $white       : #FFFFFF;
+$white       : #FFFFFF;
```
Comment (andylouisqin) [inline app/assets/stylesheets/foundation_and_overrides.scss:99]: this is already defined in CSS as `white`

File: app/assets/stylesheets/static_pages.scss (hunk 1 above)
Comment (andylouisqin) [inline app/assets/stylesheets/static_pages.scss]: should be `$ashby-gray;` ;)

File: app/assets/stylesheets/static_pages.scss (hunk 2 above)
Comment (andylouisqin) [inline app/assets/stylesheets/static_pages.scss]: same

File: app/assets/stylesheets/static_pages.scss [hunk 5]
```diff
@@ -4,6 +4,62 @@
...
+
+  .question {
+    margin-right: 10px;
+    float: left;
+  }
+
+  .input-form {
+    float: left;
+    width: 200px;
+    border-bottom: 3 px solid ashby-gray;
```
Comment (andylouisqin) [inline app/assets/stylesheets/static_pages.scss]: should be `3px` and `$ashby-gray`

**Comment by shannonchu** [comment]
Mad lib formatting and styling done.  @lilshim @andylouisqin  tried to add responsiveness, but I'm sure it needs cleaning up!
Added mask for phone field.

This I still don't know how to fix: "When you click create account and the modal shows up, if you scroll down it shows the rest of the page when it's supposed to just stay fixed. But when "position: fixed;" is added to the full-modal styling, when if you are scrolled down on the landing page and then click create account, the modal only shows partially."

File: app/views/users/registrations/edit_profile.html.erb [hunk 6]
```diff
@@ -20,7 +20,7 @@
 
   <div class="field">
     <%= f.label :phone %>
-    <%= f.text_field :phone, autofocus: true, :value => @user.phone %>
+    <%= f.text_field :phone, id: "phone", autofocus: true, :value => @user.phone %>
```
Comment (andylouisqin) [inline app/views/users/registrations/edit_profile.html.erb]: Let's try to steer away from using `id`'s! Here are a few reasons why: http://oli.jp/2011/ids/

**Pull request merged**